distribution/docker/src/docker/Dockerfile

```diff
@@ -169,15 +168,10 @@ RUN set -e ; \\
     rm "\${TINI_BIN}.sha256sum" ; \\
     mv "\${TINI_BIN}" /rootfs/bin/tini ; \\
     chmod +x /rootfs/bin/tini ; \\
-    curl --retry 10 -L -o /rootfs/bin/busybox \\
-      "https://busybox.net/downloads/binaries/\${BUSYBOX_VERSION}-defconfig-multiarch-musl/busybox-\${BUSYBOX_ARCH}" ; \\
-    chmod +x /rootfs/bin/busybox
-
-# Add links for most of the Busybox utilities
-RUN set -e ; \\
-    for path in \$( /rootfs/bin/busybox --list-full | grep -v bin/sh | grep -v telnet | grep -v unzip); do \\
-      ln /rootfs/bin/busybox /rootfs/\$path ; \\
-    done
+    curl --retry 10 -L -O \\
+      "https://github.com/docker-library/busybox/raw/\${BUSYBOX_COMMIT}/stable/musl/busybox.tar.xz" ; \\
```

Comment:
I think we need a comment explaining what's going on here, because this looks hacky compared with downloading from the official site.

Reply:
Also I don't supposed you tried with busybox 1.33.0?

Reply:
1.32 is the latest "stable" release.

Reply:
Added a clarifying comment.

Reply:
I'm a little concerned about how fiddling it will be to update those commit IDs, but let's go with it and see how we get on.

Reply:
Yeah, it's annoying they don't use tags in their repository. They just have branches per architecture.